### ISSUE TITLE
PDF top and bottom margins

### ISSUE THREAD

**Issue by paulbombarde** [qjebbs/vscode-markdown-extended#8]
Hello,

The pdf exported files does not have any top and bottom margins, making them kind of akward when printed.

From the [PhantomJS documentation](http://phantomjs.org/api/webpage/property/paper-size.html), it seems to be tunable, but I can't find a way to specify them through the plugin. May be a generic way to set PhantomJS properties could be nice feature ?

Best regards,
Paul

**Comment by paulbombarde** [comment]
This works great :+1: .
Just to be a little hard, I would really like to be able to set some default values that would be used for document not specifying the metadata...

**Comment by qjebbs** [comment]
Yes, it uses the dependency `html-pdf`'s default. If you think it's not appropriate, we can discuss a better defaults then.

**Comment by qjebbs** [comment]
please try updated plugin with default 1cm border.

**Comment by paulbombarde** [comment]
I tried it today. It's great for me and I think it should be better for everybody. The previous no-margins default was really bad.

**Comment by qjebbs** [comment]
Good to hear. I usually export to html first, and print it to PDF in chrome.
